modules/uninstall-loki-operator.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-uninstall.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="uninstall-loki-operator_{context}"]
+= Uninstalling Loki
+
+.Prerequisites
+
+* You have administrator permissions.
+* You have access to the *Administrator* perspective of the {product-title} web console.
+* If you have not already removed the {clo} and related resources, you must remove references to the LokiStack from the `ClusterLogging` custom resource.
```

Comment:
```suggestion
* If you have not already removed the {clo} and related resources, you have removed references to LokiStack from the `ClusterLogging` custom resource.
```
- Consider changing the wording to make parallel with the other prereqs. Also "must" is already implied by the fact that this is in a "Prerequisites" section.
- Not sure if "the" is needed before LokiStack?

Reply:
Perfect, thank you!